File: Difficulty: Medium/Shortest Job first/shortest-job-first.cpp
// User function Template for C++

//Back-end complete function Template for C++

class Solution {
  public:
    long long solve(vector<int>& bt) {
        // code here
        int n = bt.size();
        int ans = 0;
        sort(bt.begin(), bt.end());
        int wt = bt[0];
        
        for(int i = 1;i<n;i++) {
            ans+=wt;
            wt+=bt[i];
            
        }
        return ans/n;

        
    }
};

File: Difficulty: Medium/Shortest Job first/shortest-job-first.java
class Solution {
    static int solve(int bt[]) {
        // code here
        Arrays.sort(bt);
        int n = bt.length;
        int wt = 0;
        int totalTime = 0;
        for(int i = 1;i<n;i++) {
            wt+=bt[i-1];
            totalTime+=wt;
        }
        double avg = ((double)totalTime/(double)n);
        return (int)Math.floor(avg);
    }
}
